### DISCUSSION TITLE
container switched to unhealty after adding certificates

### DISCUSSION THREAD

**Discussion by mamema** [netdata/netdata#17025] [Q&A] [answered]
Hi,

from a healthy container to unhealthy after adding these lines according to the documentation
        ssl key	= /etc/netdata/ssl/key.pem	
        ssl certificate	= /etc/netdata/ssl/cert.pem
        tls version = 1.3
        tls ciphers = TLS_AES_256_GCM_SHA384:TLS_CHACHA20_POLY1305_SHA256:TLS_AES_128_GCM_SHA256

ther cert is from letsencrypt and is for internal purposes (zero trust) internet-facing the app is behind a reverse proxy.
Certs are valid an working in other containers.

netdata itself seems to working to, runs now via https, it's just because of those lines the container switches to unhealthy
asking here before i'm filing a bug report.
Any hints?

**Comment by ilyam8** [comment, marked as answer]
Hey, @mamema, see [health.sh](https://github.com/netdata/netdata/blob/master/packaging/docker/health.sh). To resolve the issue you need to set the `NETDATA_HEALTHCHECK_TARGET` variable to any of these 2 values:

- `cli`
- `https://localhost:19999/api/v1/info`


e.g.:

```bash
docker run -d -e NETDATA_HEALTHCHECK_TARGET=cli ...
```

**Comment by mamema** [comment]
setting the value to cli solved it, thanks
